airflow/docker_image.go

```diff
@@ -69,6 +69,12 @@ func (d *DockerImage) Build(dockerfile, buildSecretString string, buildConfig ai
 		args = append(args, "--no-cache")
 	}
 
+	if len(buildConfig.Labels) > 0 {
+		for _, label := range buildConfig.Labels {
+			args = append(args, "--label", label)
+		}
+	}
```

Comment:
```suggestion
	for _, label := range buildConfig.Labels {
		args = append(args, "--label", label)
	}
```
left over nit: we don't require the if clause now with the for loop